sdc/tests/tests_perf/generator.py

```diff
@@ -0,0 +1,104 @@
+import time
+from sdc.io.csv_ext import to_varname
+from typing import NamedTuple
+
+
+class TestCase(NamedTuple):
+    name: str
+    params: str
+    size: int = None
+    call_expression: str = None
+
+
+def gen(cases, method, class_add, typ, prefix=''):
+    for params in cases:
+        test_params = TestCase(*params)
+        name = test_params.name
+        if test_params.params:
+            name += "_" + test_params.params
```

Comment:
Wouldn't you like to use formatted strings instead of concatenating via `+`?

Reply:
@densmirn Is not + shorter and more clear than "_{}".format(...) or f"_{test_params.params}" ?

Reply:
I propose to do other way: to join `type`, `prefix`, 'name' and 'parameters' in one place with `'_'.join(...)` where we get `func_name`.